src/distance.cpp

```diff
@@ -0,0 +1,81 @@
+#include "attitude-utils.hpp"
+#include "style.hpp"
+#include "camera.hpp"
+#include "distance.hpp"
+#include <math.h>
+
+namespace found {
+    class SphericalDistanceDeterminationAlgorithm {
```

Comment:
You should change the below headers to not be enclosed in the class (still in namespace). In order to change all of this then, you're going to have change the definition format. (You don't need to specify what's public and what's private).

Every method should now be declared like the following:
`returnType ClassName::MethodName(params...)`

As an example:
`distFromEarth SphericalDistanceDeterminationAlgorithm::Run(char *image, Points &p, int imageWidth, int imageHeight)`

Also, do not indent within a namespace